taxi/forms.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+from django import forms
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "first_name",
+            "last_name",
+            "license_number",
+        )
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    length = 8
+
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_num = self.cleaned_data["license_number"]
+        first_char = license_num[:3]
+        last_char = license_num[3:]
+
+        if (len(license_num) != DriverLicenseUpdateForm.length
+                or first_char.upper() != first_char
+                or not first_char.isalpha()
+                or not last_char.isnumeric()):
+            raise ValidationError("Data is not correct")
+        return license_num
```

Comment:
too hard to read, too many != and "not" comparisons
it's better either use "not" before the whole statement of do like this:
"if a and b and c": return num
else: raise error